### PR TITLE
Create a doc index page

### PR DESCRIPTION
**Pull request by mfielding**
This is a placeholder index page, redirecting to the user guide for now.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Toolkit for Bare Metal Service/</title>
```
Comment (bpj146) [inline docs/index.html:3]: Is that trailing slash on "Service/" supposed to be there?

Comment (mfielding) [reply docs/index.html]: Nope, fixed.

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Toolkit for Bare Metal Services</title>
```
Comment (bpj146) [inline docs/index.html:3]: Sorry I didn't notice this last time (I didn't get enough sleep last night): "Services" should be "Solution"

Comment (mfielding) [reply docs/index.html]: Updated :-)

**Pull request merged**